templates/tools/dockerfile/interoptest/grpc_interop_dart/Dockerfile.template

```diff
@@ -16,5 +16,8 @@
 
   FROM google/dart:latest
 
+  # Upgrade Dart to version 2.
+  RUN apt-get update && apt-get upgrade -y dart
```

Comment:
nit: this can also be done by setting the tag in `FROM google/dart:VERSION_YOU_WANT`. Leaving up to you.

Reply:
If the latest `google/dart` image doesn't have Dart 2, then I'm not sure if any version does. I believe we have to update Dart manually until the Dockerimage base is updated.